scalafmt-dynamic/src/main/scala/org/scalafmt/dynamic/ScalafmtDynamic.scala

```diff
@@ -169,6 +169,8 @@ final case class ScalafmtDynamic(
             ScalafmtDynamicError.CannotDownload(configPath, v, None)
           case DownloadUnknownError(v, cause) =>
             ScalafmtDynamicError.CannotDownload(configPath, v, Option(cause))
+          case InvalidVersionError(v, cause) =>
+            ScalafmtDynamicError.CannotDownload(configPath, v, None)
```

Comment:
```suggestion
            ScalafmtDynamicError.CannotDownload(configPath, v, Some(cause))
```

Though we ignore the third arg of CannnotDownload, we might refer to the cause field later
https://github.com/scalameta/scalafmt/issues/1521#issuecomment-541578400